ok-marketplace-common/src/commonMain/kotlin/models/MkplAd.kt

```diff
@@ -22,4 +22,8 @@ data class MkplAd(
         permissionsClient = permissionsClient.toMutableSet(),
     )
 
+    companion object {
+        val NONE get() = MkplAd()
```

Comment:
```suggestion
        private val NONE = MkplAd()
```
это же для сравнения. Его нужно отдельным методом делать. Незачем плодить пустые объекты